resources/views/manager/job-builder-details.html.twig

```diff
@@ -0,0 +1,26 @@
+{% extends "layouts/base" %}
+
+{% block title %}{{ title }}{% endblock %}
+
+{% block body %}
+
+    {% include "common/alert" %}
+
+    {% include "common/goc" %}
+
+    {% include "common/header" %}
+
+    <a id="skipLink"></a>
+
+    <div
+        data-clone=""
+        id="job-builder-details"
+        data-job-id="{{jobId}}"
+    >
+    </div>
+
+    <script async defer
```

Comment:
Good call, but should put this in the `scripts` block that's in the base layout template.